src/plugins/plugin-player.ts

```diff
@@ -6,6 +6,15 @@
  * https://opensource.org/licenses/MIT.
  */
 
+export interface PlayerAPI {
+  get(): any;
+  set(value: any): any;
+  opponent?: {
+    get(): any;
+    set(value: any): any;
+  };
+}
+
```

Comment:
Thanks! Looks like converting `plugin-player` to Typescript introduced some errors.

Try declaring `result` on line 34 with type `PlayerAPI` and including the get and set properties directly:

```ts
    let result: PlayerAPI = {
      state,
      get: () => {
        return data.players[ctx.currentPlayer];
      },
      set: value => {
        return (state[ctx.currentPlayer] = value);
      },
    };
```

Reply:
Should be fixed now